ddl/partition.go

```diff
@@ -2373,20 +2398,25 @@ func (w *worker) onReorganizePartition(d *ddlCtx, t *meta.Meta, job *model.Job)
 			// For available state, the new added partition should wait its replica to
 			// be finished, otherwise the query to this partition will be blocked.
 			count := tblInfo.TiFlashReplica.Count
-			needRetry, err := checkPartitionReplica(count, addingDefinitions, d)
+			needRetry, err := checkPartitionReplica(count, count, addingDefinitions, d)
```

Comment:
So if one partition replica is not available, then the ddl will block wait? Why we make the change from 1 to all?

Reply:
I wanted to avoid the situation when a table has 3 replicas and after the DDL the new partitions only has 1 without any warnings etc.
If that is OK and expected, then I can remove the check for all and keep it to only check for one replica.

Reply:
@hehechen I think we can take partition process into account in current code base. It means given a 3 replica table which has 1 partition, if only 1 replica is available since then, we can see available=true and progress=0.3333.
If I am right, then I think the both the original "1 replica" policy or the "all replica" policy in this PR is acceptable, while the best way is somehow reuse the function in ddl_tiflash.go. So what do you think about this?

Reply:
I think It is OK to keep checking only one replica, because when only one of the three replicas is ready, the `progress` will become 0.33.